json_parser6.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+class JSON:
+    def json_read(self,path):
+        f=open(path,'r')
+        data=f.read()
+        f.close
+        x=json.loads(data)
+        return x
+
+if __name__ == "__main__":
+    
+    j=JSON()
+
+    path="C:\\Users\\Admin\Desktop\\check-in\\automation-frameworks\\json_parsers\\jsoninput1.txt"
```

Comment:
Please  remove the path dependency and provide the relative path from your project directory